src/brasil/gov/tiles/tiles/templates/nitf.pt

```diff
@@ -25,13 +25,13 @@
       <tal:image
           condition="python:field['id'] == 'image'"
           define="scale python:field.get('scale', 'large');
-                  position python:field.get('position', '');">
-        <a class="imag" tal:attributes="href view/getURL;
+                  position python:field.get('position', '');
+                  scales view/@@images;
```

Comment:
I think the `@@images` view must be called from the `context` and not from the `view`.

Reply:
We're using the same logic applied to https://github.com/collective/collective.nitf/blob/7160c02a743c4c162a91e5310934b28f26b49b53/src/collective/nitf/tiles/nitf.pt#L34. Should be really changed?

Reply:
I have no idea.